torch/_functorch/eager_transforms.py

```diff
@@ -359,9 +364,7 @@ def jacrev(func: Callable, argnums: Union[int, Tuple[int]] = 0, *, has_aux=False
         chunk_size (None or int): If None (default), use the maximum chunk size
             (equivalent to doing a single vmap over vjp to compute the jacobian).
             If not None, then compute the jacobian :attr:`chunk_size` rows at a time
-            (equivalent to doing multiple vmap over vjp).
-            Note that :attr:`chunk_size=1` is equivalent to computing the jacobian
-            row-by-row with a for-loop. If you run into memory issues computing
+            (equivalent to doing multiple vmap over vjp). If you run into memory issues computing
```

Comment:
nit: I'd add an extra line about 1 for completely. Redundancy in the note is fine. So:

chunk_size (None or int): If None (default), use the maximum chunk size
            (equivalent to doing a single vmap over vjp to compute the jacobian).
            If 1, then compute the jacobian row-by-row with a for-loop.
            If not None and > 1, then compute the jacobian :attr:`chunk_size` rows at a time
            (equivalent to doing multiple vmap over vjp).